pkg/network/tracer/connection/kprobe/config.go

```diff
@@ -74,27 +74,35 @@ func enabledProbes(c *config.Config, runtimeTracer bool) (map[probes.ProbeFuncNa
 		enableProbe(enabled, probes.InetBindRet)
 		enableProbe(enabled, probes.UDPSendPage)
 		enableProbe(enabled, probes.UDPSendPageReturn)
-
+		if kv >= kv470 || runtimeTracer {
+			enableProbe(enabled, probes.UDPRecvMsg)
+		} else if kv > kv410 {
+			enableProbe(enabled, probes.UDPRecvMsgPre470)
+		} else {
+			enableProbe(enabled, probes.UDPRecvMsgPre410)
+		}
+		enableProbe(enabled, selectVersionBasedProbe(runtimeTracer, kv, probes.UDPRecvMsgReturn, probes.UDPRecvMsgReturnPre470, kv470))
```

Comment:
I think you always want `probes.UDPRecvMsgReturn` and `probes.UDPv6RecvMsgReturn` if `runtimeTracer`

Reply:
`selectVersionBasedProbe` will always select that if `runtimeTracer` is `true`.

Reply:
Ah yes, I forgot that behavior.